.github/test_dependencies.yaml

```diff
@@ -56,5 +56,5 @@ envtests:
   # Because of a bug that was introduced in Kong EE 3.5 (https://konghq.atlassian.net/browse/KAG-3699),
   # we need to stick to 3.4 in order to make our KongVault validation tests stable.
   # This version should be bumped to the current one once the bug is fixed.
-  # renovate: datasource=docker depName=kong/kong-gateway versioning=docker
-  kong-ee: '3.6.1.1'
+  # PLEASE DO NOT BUMP THIS VERSION BEFORE KAG-3699 IS FIXED. renovate: datasource=docker depName=kong/kong-gateway versioning=docker
```

Comment:
So if I understand correctly this will not match the regex from https://github.com/Kong/kubernetes-ingress-controller/blob/a6511bd7be9cc47e5de40e90978ebc24051b1f5f/renovate.json#L24-L26 which in turn won't trigger the PRs with update to be created right?

This should do it I guess. Another option I thought about was to use https://docs.renovatebot.com/configuration-options/#allowedversions in renovate's config but perhaps that would be an overkill.

Reply:
Yeah, that was my idea behind putting this string in here - it should no longer match the regex. 👍